### PR TITLE
Fabo/update ledger lib

### PR DESCRIPTION
**Pull request by faboweb**
Updated ledger library to the latest version

### CODE REVIEW

File: src/cosmos-ledger.ts [hunk 1]
```diff
@@ -212,3 +215,16 @@ export const checkAppMode = (testModeAllowed: Boolean, testMode: Boolean) => {
     )
   }
 }
+
+// doesn't properly work in ledger-cosmos-js
+function getBech32FromPK(hrp, pk) {
+  if (pk.length !== 33) {
+    throw new Error('expected compressed public key [31 bytes]')
```
Comment (jbibla) [inline src/cosmos-ledger.ts]: what does this error message mean?

Comment (faboweb) [reply src/cosmos-ledger.ts]: I copied that from Juan
```suggestion
    throw new Error('The private key needs to be of exactly 33 bytes length to be used for a Cosmos address.')
```

Comment (jbibla) [reply src/cosmos-ledger.ts]: ok that's better - but is the error actionable for the user? if a user sees this error, what are they to do?

Comment (faboweb) [reply src/cosmos-ledger.ts]: This should never be user facing

Comment (jbibla) [reply src/cosmos-ledger.ts]: perfect!

**Pull request merged**